src/App.jsx

```diff
@@ -57,7 +57,7 @@ export function App() {
 							/>
 						}
 					/>
-					<Route path="/list" element={<List data={data} />} />
+					<Route path="/list/:path/:path" element={<List data={data} />} />
```

Comment:
Another way we could write the path attribute in the `Route` element would be with a template literal which uses `listPath` state (`\`/list/${listPath}\``), but I don't really know which one would be the best practice. Just a thought in case anyone can chime in, no need to change it.